templates/hoover-migrate.nomad

```diff
@@ -29,7 +29,7 @@ job "hoover-migrate" {
             SECRET_KEY = {{.Data.secret_key}}
           {{- end }}
           {{- range service "hoover-pg" }}
-            HOOVER_DB = postgresql://hoover:hoover@{{.Address}}:{{.Port}}/hoover
+            HOOVER_DB = postgresql://search:search@{{.Address}}:{{.Port}}/search
```

Comment:
Yep. We agreed to rename the pg user and database to match the ones from docker-setup. Because it makes the migration easier and why not.